smallrye-reactive-messaging-mqtt/src/test/resources/mosquitto-tls/client/client.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please add a README with the generation instructions. I've been too many times bitten by expiration.

Reply:
Where should I put the README?

Reply:
In sec/test/resources

Reply:
Ok, I put it under the mosquitto-tls directory.